src/content/peerconnection/multiple/js/test.js

```diff
@@ -26,9 +26,12 @@ test('PeerConnection multiple sample', function(t) {
   })
   .then(function() {
     t.pass('got media');
-    return driver.findElement(webdriver.By.id('callButton')).click();
+    return driver.wait(function() {
+      return driver.findElement(webdriver.By.id('callButton')).isEnabled();
+    });
```

Comment:
Did this on purpose, i.e. want to see which is more stable, isEnabled or elementIsVisible.